README.md

```diff
@@ -973,6 +972,61 @@ Learn more about class-transformer and how to handle more complex object constru
 This behaviour is enabled by default.
 If you want to disable it simply pass `useClassTransformer: false` to createExpressServer method.
 
+## Auto validating action params
+
+Sometimes parsing a json object into instance of some class is not enough. 
+E.g. `class-transformer` doesn't check whether the property's types are correct, so you can get runtime error if you rely on TypeScript type safe. Also you may want to validate the object to check e.g. whether the password string is long enough or entered e-mail is correct.
+
+It can be done easily thanks to integration with [class-transformer-validator][7] which internally depends on [class-validator][6]. So first of all, you have to install the modules:
+```
+npm install class-validator --save
+npm install class-transform-validator --save
+```
+
+Then simply specify a `useParamValidator: true` option on application bootstrap:
+```typescript
+import "reflect-metadata";
+import { createExpressServer } from "routing-controllers";
+
+createExpressServer({
+    useParamValidator: true
+}).listen(3000);
+```
+
+Now you need to define the class which type will be used in controller method params. Decorate the properties with appropriate validators.
+```typescript
+export class User {
+
+    @IsEmail()
+    email: string;
+
+    @MinLength(6)
+    password: string;
+
+}
+```
+If you haven't used class-validator yet, you can learn how to use the decorators and handle more complex object validation [here][6].
+
+Now, if you have specified a class type, your action params will be not only an instance of that class (with the data sent by a user) but they will be validated too, so you don't have to worry about eg. incorrect e-mail or too short password and manual checks every property in controller method body.
+```typescript
+@Controller()
+export class UserController {
+
+    @Post("/login/")
+    login(@Body() user: User) {

```

Comment:
probably it also a good to have ability to control validation per-use-case, e.g. `@Body({ validate: false|true })`